### PR TITLE
Create Contact form

### PR DESCRIPTION
**Pull request by codedbystuart**
## Changed Files
* index.html
* styles.css

## Features
Created contact form and used Formspree service to make the contact form functional.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,248 +1,264 @@
...
+      <ul>
+        <li class="list-group">
+          <a href="https://@Maxwell_Chuks67"><img src="./images/Twitter.png" alt="Twitter Logo" /></a>
+          <a href="https://www.linkedin.com/in/chukwuemeka-orji-50b766241/"><img src="./images/Linkedin.png"
+              alt="Linkedin Logo" /></a>
+          <a href="https://github.com/Maxwell011"><img src="./images/Medium.png" alt="Medium Logo" /></a>
+          <a href="https://github.com/Maxwell011"><img src="./images/Vector.png" alt="GitHub Logo" /></a>
+          <a href="https://github.com/Maxwell011"><img src="./images/Angelist.png" alt="Angelist Logo" /></a>
+        </li>
+      </ul>
```
Comment (BenMukebo) [inline index.html]: - [ ] According to the project requirements, Flexbox must be used to layout the elements. Kindly make sure the ul > li content is aligned with flexbox. This gives us a lot of options to align all our elements, plus, it's responsive. 👍🏻

File: index.html [hunk 2]
```diff
@@ -1,248 +1,264 @@
...
+      <p class="email">chukwuemekaorji678@gmail.com</p>
```
Comment (BenMukebo) [inline index.html]: - [ ] Kindly add this missing bar 👇🏻

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @codedbystuart,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- No linter errors. ✔️ 
- GitHub flow was correctly used.  ✔️ 
- All validations described in the project description are working on form submission.🥇 
- The form is submitted successfully with valid input values. 👏🏻 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_


- [ ] It will be nice to add explicit steps to clone and run your project locally, as this will be of help to non-technical users. You can put the steps to clone and run the project in a new Getting Started section under the project's description. This includes `git clone URL` , `cd project_name`, etc 🙃  Please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.
- The README file It is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. It will make your project stand out from a bunch of others.

- [ ] Make sure the content is aligned with flexbox. since this gives us a lot of options to align all our elements, plus, it's responsive.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — APPROVED** [review]
Hi @codedbystuart , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

YesYouAreGIF (2)

## Highlights:
✅ Correct use of Github workflow
✅ No linter error
✅ Good design


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**